### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by VDB-Bram**
## Description

- fix typos
- center logo
- add reference to cite when using PredSim
- update section on using the issues and discussion section
- add section on replicating results
 
## Motivation and Context

For users it was not clear how to cite PredSim.
Information on using the discussion section was missing.
For users it was not clear how to replicate results.

## How Has This Been Tested?

Not applicable

## Suggested tests for reviewers

Proofread. Also look at it visually by [going to the brach](https://github.com/KULeuvenNeuromechanics/PredSim/tree/update-readme-sept) instead of only looking at 'Files changed'.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,30 @@
...
-This repository is a work in progress
-- If you encounter an error or get an unexpected output, [check the issues](https://github.com/KULeuvenNeuromechanics/PredSim/issues). Please look at the existing issues before creating a new one, you might not be the first person to have this problem. Include the error messages in the issue.
-- Feel free to suggest improvements. Submit a pull request with the changes, or create an issue with a desciption of the proposed change.
+This repository is a work in progress.
+1. ⚠️ **Troubleshooting Issues**:
+   - If you encounter an error or get an unexpected output, let us know. Does the answer to your question/remark probably require code changes?
+     - Yes -> use the [issues](https://github.com/KULeuvenNeuromechanics/PredSim/issues). Please look at the existing issues before creating a new one; you might not be the first person to have this problem.
+     - No -> use the [discussions](https://github.com/KULeuvenNeuromechanics/PredSim/discussions). 
+2. 🚧 **Suggesting Improvements**:
+   - Feel free to suggest improvements. Submit a pull request with the changes, or create an issue with a description of the proposed change.
```
Comment (Lars-DHondt-KUL) [inline README.md:25]: > ..., or post it in the discussions

let's limit the issues to features that we intend to add

File: README.md [hunk 2]
```diff
@@ -80,6 +86,12 @@ This code can automatically convert an OpenSim model to the external function us
 
 A [full overview of settings](./Documentation/SettingsOverview.md) is available in the documentation.
 
+## Replicate results
+
+Results of previous publications can be replicated by specifying one of the inputs for [initializeSettings](main.m#L18):
```
Comment (Lars-DHondt-KUL) [inline README.md:91]: Also need to set subject name and osim_path

File: README.md [hunk 3]
```diff
@@ -80,6 +86,12 @@ This code can automatically convert an OpenSim model to the external function us
...
+- `'Falisse_et_al_2022'`
```
Comment (Lars-DHondt-KUL) [inline README.md:92]: add number of the reference for each

File: README.md (hunk 2 above)
Comment (VDB-Bram) [reply README.md:91]: osim_path should be fine if subject name is ok, no?

**Review by Lars-DHondt-KUL — APPROVED** [review]

**Pull request merged**